open_moreinfo.module

```diff
@@ -222,12 +222,29 @@ function open_moreinfo_get_objects($object_id, $object_type = NULL) {
           }
           else {
             $filetype = $moreinfo_object->getFileType($type);
-            if ($file = _open_moreinfo_fetch_object($local_id, $type, $filetype,
-              $moreinfo[$faust_id]->$type)) {
-              //Generate a path corresponding to the downloaded image
-              $objects[$local_id][$type] =
-                file_default_scheme() . '://moreinfo/' . $type . '/' . md5($local_id) .
-                $moreinfo[$faust_id]->getFileType($type);
+
+            if (is_array($moreinfo[$faust_id]->$type)) {
```

Comment:
Nogle gange har vi arbejdet med at lave alt om til array, så man ikke skal lave det samme i koden to gange. Jeg ved ikke rigtig hvad der er det smarteste...

Reply:
Jeg er foran på tiden, så jeg giver det lige et forsøg.